components/dropdownbutton/events.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Events
+page_title: DropDownButton Events
+description: How to handle events with the DropDownButton for Blazor.
+slug: dropdownbutton-events
+tags: telerik,blazor,dropdownbutton,events
+published: True
+position: 15
+---
+
+# DropDownButton Events
+
+This article describes the DropDownButton events:
```

Comment:
- A list with a single item looks unnatural and wastes space.
- An additional heading isn't required for such a short article. Such structure makes sense when the component exposes two or more events.
```suggestion
The DropDownButton exposes an [`OnClick`](#onclick) event that you can use to initiate an action within the application.
```